### PR TITLE
feat(zipkin): add support for labeling phase duration as tags

### LINKED ISSUE #8126: Rationale for logs/annotations instead of spans?
Can you explain the rationale for using logs/annotations (krs... 10 total) instead of 5 additionnal spans? 
The performance cannot be considered top priority, as you just need to lower your sampling rate.
Is it visualized similarly in zipkin, i.e. different spans vs multiple annotations?
What about searching for annotations?
We are using Jaeger as backend, and feel it will be hard to investigate any timing issues in kong plugins, I don't think I can search for the "time difference between annotations >= XX milliseconds" anywhere even directly in ES storage of Jaeger.
Maybe you will answer that this is more of a metric type of search.

This is what a kong-zipkin-plugin trace looks like in Jaeger-ui, you have to expand to see them, they do not appear in the timeline, so we were a bit underwhelmed I must say:
L_B8D5 tmp

Is it worth a PR with a configuration choice between multiple spans vs annotations?

Thanks for your answers.
Regards.

### PR DESCRIPTION
**Pull request by mayocream**
### Summary

Add support to set durations as span tags.

### Checklist

- [x] The Pull Request has tests
- [x] There's an entry in the CHANGELOG
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Issue reference

Fix #8126
Reimplement #8264

FTI-4507

### CODE REVIEW

File: kong/plugins/zipkin/handler.lua [hunk 1]
```diff
@@ -292,43 +301,60 @@ function ZipkinLogHandler:log(conf) -- luacheck: ignore 212
...
-      ngx_ctx.KONG_PREREAD_ENDED_AT and ngx_ctx.KONG_PREREAD_ENDED_AT * 1000
-      or proxy_finish_mu
-    proxy_span:annotate("kpf", preread_finish_mu)
+
+    if conf.phase_duration_flavor == "annotations" then
+      local preread_finish_mu =
+        ngx_ctx.KONG_PREREAD_ENDED_AT and ngx_ctx.KONG_PREREAD_ENDED_AT * 1000
+        or proxy_finish_mu
+      proxy_span:annotate("kpf", preread_finish_mu)
+
```
Comment (windmgc) [inline kong/plugins/zipkin/handler.lua:353]:
```suggestion
```

**Review by windmgc — APPROVED** [review]
Approved with a nitpick

**Comment by hbagdi** [comment]
@mayocream Can you please add a changelog entry and open a corresponding documentation PR?

**Comment by mayocream** [comment]
> @mayocream Can you please add a changelog entry and open a corresponding documentation PR?

Ahh, I already added the changelog entry; I just forgot to update the PR checklist.
Also created the docs PR: https://github.com/Kong/docs.konghq.com/pull/4882

**Pull request merged**